source/fundamentals/crud/read/limit.txt

```diff
@@ -0,0 +1,117 @@
+            .append("title", "Atlas Shrugged").append("length", 1088).append("author", "Rand"),
+        new Document().append("_id", 4)
+            .append("title", "Infinite Jest").append("length", 1104).append("author", "Wallace"),
+        new Document().append("_id", 5)
+            .append("title", "Cryptonomicon").append("length", 918).append("author", "Stephenson"),
+        new Document().append("_id", 6)
+            .append("title", "A Dance with Dragons").append("length", 1104)
+            .append("author", "Martin")
+    ));
+
+The following example queries the collection to return the top three
+longest books. It matches all the documents with the query, applies
+a ``sort`` on the ``length`` field to return books with longer lengths before
+books with shorter lengths, and applies a ``limit`` to return only ``3`` results:
+
+.. code-block:: java
+
+    // define a cursor that will return the first 3 sorted items
+    MongoCursor<Document> cursor = collection.find()
+        .sort(descending("length"))
+        .limit(3)
+        .iterator();
+    // print out items
+    try {
+      while (cursor.hasNext()) {
+        System.out.println(cursor.next().toJson());
+      }
+    // close the cursor
+    } finally {
+      cursor.close();
+    }
+
+
+The code example above outputs the following three documents, sorted by
+length:
+
+.. code-block:: javascript
+
+   { "_id": 2, "title": "Les Misérables", "author": "Hugo", "length": 1462 }
+   { "_id": 6, "title": "A Dance With Dragons", "author": "Martin", "length": 1104 }
+   { "_id": 4, "title": "Infinite Jest", "author": "Wallace", "length": 1104 }
+
+.. note::
+
+   The order in which you call ``limit`` and ``sort`` does not matter
+   because the driver reorders the calls to apply the sort first and the
+   limit after it. The following two calls are equivalent:
+
+   .. code-block:: java
+
+      collection.find().sort(descending("length")).limit(3);
+      collection.find().limit(3).sort(descending("length"));
+
+
+To see the next three longest books, append the ``skip()`` method, passing
+the number of documents to skip over to the previous code snippet's call to
+``find()``:
+
+.. code-block:: java
+
+    MongoCursor<Document> cursor = collection.find()
+      .sort(ascending("length")).limit(3).skip(3).iterator();
+    try {
+      while(cursor.hasNext()) {
+        System.out.println(cursor.next().toJson());
+      }
+    } 
+    finally {
+      cursor.close();
+    }
+
+This operation returns the documents that describe the fourth through sixth
+longest books:
+
+.. code-block:: javascript
+
+   { "_id": 3, "title": "Atlas Shrugged", "author": "Rand", "length": 1088 }
+   { "_id": 5, "title": "Cryptonomicon", "author": "Stephenson", "length": 918 }
+   { "_id": 1, "title": "The Brothers Karamazov", "author": "Dostoyevsky", "length": 824 }
+
+.. note::
+
+   These results are in JSON format due to the ``toJson()`` method used in the example above.
```

Comment:
Couple of things here.
1. Passive voice, suggest reword to active
2. "returned in a Java string format" isn't strictly true here, because we're actually printing the documents and they're turned into a string either way -- toJson() just turns the objects into a different format first, so they use a different toString().
3. Is there a reason why we use toJson() here anyway? I suppose it is a bit more readable than the default toString method, but I wonder if we're better off displaying the results that way instead if we need to include a note anyway to explain what's going on with toJson().

I'd suggest a rewording to something like this, but feel free to reword however you see fit -- this is just a suggestion I came up with on the fly:
```
These results appear in JSON format in this example because
we use the ``toJson()`` method to transform the results into
a JSON format that prints as a JSON string. If you omit the
``toJson()`` method call, the results instead print using the
``toString()`` method of the
:doc:`data format </fundamentals/data-formats>` used to
represent objects in your ``MongoCollection``, which
defaults to ``Document``.
```

Reply:
To your 3rd point, I used it because of readability, but it seems like it might simplify things a bit to simply omit ``toJson()`` and this note altogether, and display the documents as they're originally printed.